recipes/hsm/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_executable(example example.cpp)
```

with the suggestion above

```suggestion
add_executable(example example.cpp)
target_link_libraries(example hsm::hsm)
```

---

+1